packages/frontend/editor-ui/src/components/Folders/MoveToFolderModal.test.ts

```diff
@@ -45,7 +66,77 @@ const TEST_WORKFLOW_RESOURCE = {
 	parentFolderId: 'test-parent-folder-id',
 };
 
-let uiStore: ReturnType<typeof mockedStore<typeof useUIStore>>;
+let uiStore: MockedStore<typeof useUIStore>;
+let settingsStore: MockedStore<typeof useSettingsStore>;
+let credentialsStore: MockedStore<typeof useCredentialsStore>;
+let workflowsStore: MockedStore<typeof useWorkflowsStore>;
+let foldersStore: MockedStore<typeof useFoldersStore>;
+let projectsStore: MockedStore<typeof useProjectsStore>;
+
+const personalProject = createProjectListItem('personal');
+const teamProjects = Array.from({ length: 3 }, () => createProjectListItem('team'));
+const projects = [personalProject, ...teamProjects];
+const homeProject = createProjectSharingData();
+
+const enableSharing = {
+	enterprise: {
+		sharing: true,
+	},
+} as FrontendSettings;
+
+const createCredential = (overrides = {}): ICredentialsResponse => ({
+	id: faker.string.alphanumeric(10),
+	createdAt: faker.date.recent().toISOString(),
+	updatedAt: faker.date.recent().toISOString(),
+	type: 'generic',
+	name: faker.lorem.words(2),
+	sharedWithProjects: [],
+	isManaged: false,
+	homeProject,
+	...overrides,
+});
+
+const readableCredential = createCredential({
+	scopes: ['credential:read'],
+});
+const readableUsedCredential: IUsedCredential = {
+	id: readableCredential.id,
+	name: readableCredential.name,
+	credentialType: readableCredential.type,
+	currentUserHasAccess: true,
+	homeProject,
+	sharedWithProjects: [],
+};
+
+const shareableCredential = createCredential({
+	scopes: ['credential:share', 'credential:read'],
+});
+const shareableUsedCredential: IUsedCredential = {
+	id: shareableCredential.id,
+	name: shareableCredential.name,
+	credentialType: shareableCredential.type,
+	currentUserHasAccess: true,
+	homeProject,
+	sharedWithProjects: [],
+};
+
+const folder: ChangeLocationSearchResult = {
+	createdAt: faker.date.recent().toISOString(),
+	updatedAt: faker.date.recent().toISOString(),
+	id: faker.string.alphanumeric(10),
+	name: 'test',
+	homeProject,
+	tags: [],
+	workflowCount: 0,
+	subFolderCount: 0,
+	path: ['test'],
+	resource: 'folder',
+};
+
+const mockEventBus = {
+	emit: vi.fn(),
+};
+
 describe('MoveToFolderModal', () => {
```

Comment:
These tests now test this modal way more than before, making some of the E2E tests redundant - but I didn't get rid of any of them yet.